components/taupage.rst

```diff
@@ -688,3 +688,25 @@ You can build your own Taupage AMI using the code from the repository on GitHub
 In the repository you will find a configuration (config-stups-example.sh) file which you'll have to adjust to your needs.
 
 See :ref:`taupage-ami-creation` for details.
+
+
+Using Nvidia gpus with nvidia-docker
++++++++++++++++++++++
+If gpu device files are found on the host the usage of nvidia-docker is triggered automatically. This has the benefit that drivers are automatically mounted into the docker container (there is no need to install identical drivers in the container).
+
+Some requirements:
+
+  * gpu instances must be available in the zone of your stups team account (eg: eu-west-1)
```

Comment:
* gpu -> GPU
* stups -> STUPS

the list indentation looks wrong, did you test with a local HTML build?